COC_PyBay.md

```diff
@@ -0,0 +1,76 @@
+#Code of Conduct
```

Comment:
Meta comment: I suggest putting all three documents in a directory, `code-of-conduct`. This makes it easier to see them together after the PR is merged.

Reply:
done